examples/glueviz_wcs_example.ipynb

```diff
@@ -0,0 +1,208 @@
+    "glue-vispy-viewers        0.12.dev0                pypi_0    pypi\n",
+    "```\n",
+    "\n",
+    "Related issues:\n",
+    "\n",
+    "* https://github.com/spacetelescope/jdaviz/issues/25\n",
+    "* https://github.com/glue-viz/glue-jupyter/issues/112\n",
+    "\n",
+    "See also:\n",
+    "\n",
+    "* http://docs.glueviz.org/en/stable/developer_guide/linking.html"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "\n",
+    "from glue.core.data_factories import load_data\n",
+    "\n",
+    "from glue_jupyter import jglue"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Data files can be downloaded from https://stsci.app.box.com/s/hgj2yuhya3fjwx13s4b2ibuwiqbsmyvu . The example here assumes that you already downloaded the data files into the local working directory, where you are running this notebook from."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename_d1 = 'NIRISS_f150w_direct_dit1_nis_cal.fits'\n",
+    "filename_d2 = 'NIRISS_f150w_direct_dit2_nis_cal.fits'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with warnings.catch_warnings():\n",
+    "    warnings.simplefilter('ignore')  # Silence warnings\n",
+    "    dither1 = load_data(filename_d1)[0]  # 0 = SCI\n",
+    "    dither2 = load_data(filename_d2)[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Shows the data collection.\n",
+    "dither1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Shows the data components for SCI extension.\n",
+    "dither1.components"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we create a `glue-jupyter` app and add the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app = jglue()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.add_data(dither1)\n",
+    "app.add_data(dither2);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Having loaded both SCI extensions from different dithers and know their WCS components, we can now link them.\n",
+    "\n",
+    "**TODO: What does this really do? Even with this added, when I pan in first viewer, the second one does not auto-pan.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.add_link(dither1, 'Right Ascension', dither2, 'Right Ascension')\n",
```

Comment:
I thought this was the right way to link the data together. But even when this is executed, panning `viewer1` below does not auto-pan display in `viewer2`.

In contrast, when I select a subset on `viewer1`, the selection box shows up on both viewers and appear to be sync-ing properly.

Reply:
Just to be clear, linking the data means you can show the two datasets in the *same* viewer as overlays to each other (or e.g. as an RGB image if you have three datasets). Linking the views is different - I'll post an example of how to do this.